templates/download/desktop/thank-you.html

```diff
@@ -256,28 +316,31 @@ <h3 class="p-card--title">Select topics you're interested in</h3>
 </section>
 
 {% if version %}
-<div class="p-strip is-shallow">
-  <div class="row u-equal-height">
-    <div class="col-4 p-card--highlighted">
-      <h3>Learn how to try Ubuntu before you install</h3>
-      <p class="p-card__content">Run Ubuntu directly from either a USB stick or a DVD for a quick and easy way to experience how Ubuntu works.</p>
-      <p class="p-card__content"><a  href="/tutorials/try-ubuntu-before-you-install">Follow our simple guide</a></p>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2 class="p-heading--4"><a href="/blog/desktop">Follow our blog</a> for the latest on Ubuntu Desktop:</h2>
```

Comment:
Would it be better with `p-heading--3` here?

```suggestion
    <h2 class="p-heading--3"><a href="/blog/desktop">Follow our blog</a> for the latest on Ubuntu Desktop:</h2>
```